FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/model/OrgRules.kt

```diff
@@ -30,16 +30,27 @@ data class OrgRules constructor(
   @SerializedName("enforce_keygen_algo")
   @Expose val enforceKeygenAlgo: KeyAlgo? = null,
   @SerializedName("enforce_keygen_expire_months")
-  @Expose val enforceKeygenExpireMonths: Int? = null
+  @Expose val enforceKeygenExpireMonths: Int? = null,
+  @SerializedName("in_memory_pass_phrase_session_length")
+  @Expose val inMemoryPassPhraseSessionLength: Int? = null
 ) : Parcelable {
 
+  val inMemoryPassPhraseSessionLengthNormalized: Int?
+    get() {
+      val value = inMemoryPassPhraseSessionLength ?: 0
+      return if (value > 0) {
+        minOf(value, Int.MAX_VALUE)
```

Comment:
I think this just can be always `value` - min of max possible integer and another integer is always that another integer.